src/bidiMapper/domains/network/NetworkUtils.ts

```diff
@@ -64,3 +83,21 @@ export function bidiNetworkHeadersFromCdpFetchHeaderEntryArray(
     },
   }));
 }
+
+/** Converts from Bidi network headers to CDP Fetch domain header entries. */
+export function cdpFetchHeaderEntryArrayFrombidiNetworkHeaders(
```

Comment:
Nit: The comment is nice but the function name is confusing. Suggestion
`convertBidiHeadersToCdpFetchHeaders`

Reply:
We're using the "from" convention in other parts of the codebase. I'd rather be consistent.

E.g. inchesFromCm instead of cmToInches.

In case you're curious why this started, see https://www.joelonsoftware.com/2005/05/11/making-wrong-code-look-wrong/

I could rename/simplify the atoms if desired though e.g. BidiNetworkHeaders -> BidiHeaders. What's your call?

Reply:
I must have missed the this style for this repo. So I think 2 thinks lets remove `EntryArray` as that is insured by the TypeSystem, and use correct capitalization so - `cdpFetchHeadersFromBidiNetworkHeaders`